circle.yml

```diff
@@ -25,15 +25,52 @@ test:
   override:
     - npm run --silent lint
     - npm run --silent lint_md
-    # lint the python used for testing
     - flake8 $(git ls-files '*.py')
     - yamllint $(git ls-files '*.yml')
+
     - mkdir -p $CIRCLE_TEST_REPORTS/unit
     - npm run unit_coverage
-    - S3BACKEND=mem npm start & sleep 4 && npm run ft_test;
-    - S3BACKEND=mem npm start & sleep 4 &&
-      ENABLE_KMS_ENCRYPTION=true npm run ft_test;
-    - S3BACKEND=file S3VAULT=mem npm start & sleep 4 && npm run ft_test;
-    - rm -rf localData/* localMetadata/*
-    - S3BACKEND=file S3VAULT=mem npm start & sleep 4 &&
-      ENABLE_KMS_ENCRYPTION=true npm run ft_test;
```

Comment:
I was confused as to where the rspec and java tests are.  This PR should be against rel/6.2.1.  You will have those tests in here then as well.